src/index.js

```diff
@@ -20,6 +20,7 @@ import Schema from './models/schema'
 import Selection from './models/selection'
 import State from './models/state'
 import Text from './models/text'
+import Range from './models/range'
```

Comment:
Nope, I don't have an opinion. Just curious since it's always better to export less, but if you're using it for a valid use case that sounds good to me to keep it exported!